Description: Unescape quote in qrdelete docstring
 Unscape quotes in @qcode macro to avoid a Texinfo parsing error,
Author: Rafael Laboissiere <rafael@debian.org>
Forwarded: https://savannah.gnu.org/bugs/index.php?48722
Last-Update: 2016-08-08

diff -r f99c0b8cb67b -r 61707b6b22c6 libinterp/dldfcn/qr.cc
--- a/libinterp/dldfcn/qr.cc	Sun Aug 07 04:21:14 2016 +0100
+++ b/libinterp/dldfcn/qr.cc	Sun Aug 07 13:06:34 2016 -0300
@@ -1242,7 +1242,7 @@
 @var{R}@tie{}upper trapezoidal, return the QR@tie{}factorization of
 @w{[A(:,1:j-1), U, A(:,j:n)]},
 where @var{u} is a column vector to be inserted into @var{A}
-(if @var{orient} is @qcode{\"col\"}),
+(if @var{orient} is @qcode{"col"}),
 or the QR@tie{}factorization of @w{[A(1:j-1,:);X;A(:,j:n)]},
 where @var{x} is a row @var{orient} is @qcode{"row"}).
 The default value of @var{orient} is @qcode{"col"}.
